Directory.Build.props

```diff
@@ -56,6 +56,11 @@
     <CentralPackageTransitivePinningEnabled>true</CentralPackageTransitivePinningEnabled>
   </PropertyGroup>
 
+  <PropertyGroup Condition="'$(PgoInstrument)' == 'true'">
+    <SkipBuildingInstallers>true</SkipBuildingInstallers>
+    <PgoTerm>-pgo</PgoTerm>
+  </PropertyGroup>
+
```

Comment:
This was something I missed when merging in the dotnet/installer repo.

Reply:
This already got added into main.

Reply:
https://github.com/dotnet/sdk/commit/3aa930f2d09ef7f3e5187aa01175d9f9688b5ce9

Reply:
```suggestion
```